CIP-0100/README.md

```diff
@@ -0,0 +1,132 @@
+---
+CIP: 100?
+Title: Supercharged Native Scripts for Minting and Burning Cardano Native Tokens
+Category: Tokens
+Status: Proposed
+Authors:
+    - Niels Mündler <n.muendler@posteo.de>
+Implementors: []
+Discussions:
+    - https://github.com/cardano-foundation/cips/pulls/?
```

Comment:
```suggestion
    - https://github.com/cardano-foundation/cips/pulls/479
```

Reply:
@Ryun1 I think these indisputable items are always OK to merge immediately if the PR author has left us write permission in their branch (especially in this common case where it fixes a dead link in the header).